website/en/docs/types/utilities.md

```diff
@@ -413,6 +414,32 @@ props(promises).then(o => {
 });
 ```
 
+## `$ObjMapi<T, F>` <a class="toc" id="toc-objmapi" href="#toc-objmapi"></a>
+
+`ObjMapi<T, F>` is similar to [`ObjMap<T, F>`](#toc-objmap), the difference being the given function type `F` will be [called](#toc-call) with both key and value instead of just the value. For example:
+
+```js
+// @flow
+const o = {
+  a: () => true,
+  b: () => 'foo'
+};
+
+type ExtractReturnObjectType = <K, V>(K, () => V) => { k: K, v: V }
+function run<A, O: Object>(o: O): $ObjMapi<O, ExtractReturnObjectType> {
```

Comment:
What is the first generic parameter (`A`) for?

Reply:
I've been trying to get them to pay attention to easy PRs like this for a long time. Gave up trying.